examples/react-todos/.meteor/packages

```diff
@@ -13,5 +14,6 @@ reactive-var
 meteorhacks:npm
 cosmos:browserify
 
+static-html
```

Comment:
I mean like https://github.com/kangax/html-minifier
but I guess we Meteor doesn't do it anyway so never mind...